pkg/dockerregistry/server/repositorymiddleware.go

```diff
@@ -178,7 +178,7 @@ func newRepositoryWithClient(
 	if err != nil {
 		context.GetLogger(ctx).Error(err)
 	}
-	pullthrough, err := getBoolOption(PullthroughEnvVar, "pullthrough", false, options)
+	pullthrough, err := getBoolOption(PullthroughEnvVar, "pullthrough", true, options)
```

Comment:
We reserve the right to enable new features in a backwards compatible way (generally).  Pullthrough is as secure as our current image pull mechanism, so the original reasons for disabling this are no longer required.